javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -23,7 +23,7 @@
     @SuppressWarnings("unchecked")
     public static <T> IntMap<T> of(Map<Integer, T> original) {
         return original.isEmpty() ? (IntMap<T>) EMPTY
-                                  : new IntMap<>(original);
+                : new IntMap<>(original);
```

Comment:
Minor: please leave the original indentation in these cases.
In Idea you can configure it in:
```
File | Settings | Editor | Code Style | Java
Wrapping and Braces | Ternary operation | Align when multiline | ☑
```